guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,173 @@
+[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+= Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+
+Use this procedure to upgrade your {Project} installation from Enterprise Linux 7 to Enterprise Linux 8.
+LEAPP is the supported tool for Enterprise Linux 7 to Enterprise Linux 8 upgrades.
+
+.Prerequisites
+* {Project} {ProjectVersionPrevious} running on CentOS Linux 7.
+* {Project} installations running on {RHEL} 7 cannot upgrade to {RHEL} 8.
+You must have a {RHEL} compatible LEAPP build to upgrade.
+* Access to available repositories or a local mirror of repositories.
+* No outstanding updates.
+* {Project} installations can be upgraded to CentOS Stream 8 and a {RHEL} rebuild.
+
+.Procedure
+[start=1]
+. Configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/[@theforeman/leapp COPR Repository], which contains newer LEAPP packages than those shipped by https://wiki.almalinux.org/elevate/[AlmaLinux/ELevate], and support {Project} upgrades:
++
+----
+# curl -o /etc/yum.repos.d/theforeman-leapp.repo https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/repo/epel-7/group_theforeman-leapp-epel-7.repo
+----
+[start=2]
+. Install required packages:
++
+----
+# yum install leapp leapp-repository leapp-data-centos
+----
+
+[start=3]
```

Comment:
I am still not convinced we really need this. It looks very error-prone in the long run. Usually, when you exclude certain steps for certain built targets, the lists in asciidoc still work as expected.